demo-new/package.json

```diff
@@ -6,6 +6,7 @@
     "@babel/core": "^7.18.5",
     "@babel/preset-env": "^7.18.2",
     "@babel/preset-react": "^7.17.12",
+    "@kong/swagger-ui": "^3.26.3",
```

Comment:
This version will not be used since we link `kong/swagger-ui` on `next/4.x` branch